### PR TITLE
Add basic read hit performance tests for CAS.

### PR DESCRIPTION
**Pull request by KlaudiaJ**
Tested in 'Write-Through' and 'Write-Back' mode.

Signed-off-by: Klaudia Jablonska <klaudia.jablonska@intel.com>

### CODE REVIEW

**Review by jfckm — CHANGES_REQUESTED** [review]

File: test/functional/tests/performance/test_performance_basic.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    set_wbt_lat,
+    get_dut_cpu_physical_cores,
+)
+from test_utils.size import Unit, Size
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrize("block_size", [Size(1, Unit.Blocks4096), Size(8, Unit.Blocks4096)])
+@pytest.mark.parametrize("queue_depth", [1, 16, 32])
```
[the remark this answers is not in the document]
Comment (KlaudiaJ) [reply test/functional/tests/performance/test_performance_basic.py:44]: Its good to have less iterations (this tests take a lot of time). I was just thinking what is the probability that the performance will drop only in specific configurations and not the others.

File: test/functional/tests/performance/test_performance_basic.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                TestRun.LOGGER.error(f"The read iops ratio is below expected threshold (85%).")
+
+
+# TODO: for disks other than Intel Optane, fio ramp is needed before fio tests on raw disk
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("block_size", [Size(1, Unit.Blocks4096), Size(8, Unit.Blocks4096)])
+@pytest.mark.parametrizex("queue_depth", [1, 16, 32])
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_performance_read_hit_wb(cache_line_size, block_size, queue_depth):
```
Comment (jfckm) [inline test/functional/tests/performance/test_performance_basic.py:155]: test names are duplicated - this way they're shadowing each other and the earlier one won't run (I think)

**Review by jfckm — CHANGES_REQUESTED** [review]

File: test/functional/tests/performance/test_performance_basic.py (hunk 2 above)
Comment (KlaudiaJ) [reply test/functional/tests/performance/test_performance_basic.py:155]: They are similar but not duplicated ;)
There is 'w**t**' and 'w**b**' at the and of the name to reflect Write-Through and Writer-Back modes used in test.

**Review by jfckm — APPROVED** [review]

**Comment by KlaudiaJ** [comment]
Retest this please

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**